### PR TITLE
Milestone_1: Project Setup

### PR DESCRIPTION
**Pull request by kubilaycaglayan**
## Dear TSE :wave: 

[Heroku Link](https://kubilay-calculator.herokuapp.com/)
[Readme](https://github.com/kubilaycaglayan/calculator/blob/m1_project_setup/README.md)

###  :mag:  In development branch:

1. Initialized react app
2. Deleted unnecessary files
3. Added big.js dependency

###  :mag:  In feature(m1_calculator_setup) branch:

4. Initialized linters
5. Deployed to Heroku
6. Updated readme

With all my respect... :pray:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,5 +31,17 @@
       "last 1 firefox version",
```
Comment (bellom) [inline package.json:31]: With all my respect, we need to add `prop-types` 🙏 
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @kubilaycaglayan  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: package.json (hunk 1 above)
Comment (kubilaycaglayan) [reply package.json:31]: Thanks for the review, for some reason I missed that.

**Review by mosaaleb — APPROVED** [review]
#### Well done @kubilaycaglayan :clap:

All requirements from previous review are handled.
You can merge :+1:

Good luck with then next milestones.

##### Muhammad Ebeid

**Pull request merged**